test/models/user_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name: "Example User", email: "user@example.com", password: "foobar", password_confirmation: "foobar")
+  end
+
+  test "name should not be too long" do
+    @user.name = "a" * 51
```

Comment:
ditto, see comments I made on Oratile's PR re: test names + extracting & using constants for these constraint values like max name length